server/actions/civTurnNotificationAction.js

```diff
@@ -16,54 +16,55 @@ async function civTurnNotificationAction(request, response) {
     return response.send("");
   }
 
+  const gameName = request.body.value1;
   const steamUser = request.body.value2;
   const discordId = await getDiscordId(channel.channelId, steamUser);
 
   const userStr = discordId ? `<@${discordId}>` : steamUser;
-  const message = getRandomTurnNotifcation(userStr);
+  const message = getRandomTurnNotifcation(gameName, userStr);
```

Comment:
Thanks for the contribution! Rather than duplicate the same prefix a bunch below, lets just add it once here. Something like

```
const gamePrefix = `${gameName} : `;
const message = ...
postMessage(channel.channelId, gamePrefix + message);
```

Reply:
Great suggestion :P